src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -187,6 +214,6 @@ private function getOriginalToken(TokenInterface $token)
             }
         }
 
-        return false;
+        return;
```

Comment:
I said it's useless because in PHP a method returns `null` also if you don't call an explicit `return` or `return null`

ok for the `@return` doc, thanks